exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -31,7 +31,7 @@ public void testVerseTwo() {
     @Test
     public void testVerseThree() {
         String expectedVerseThree = "On the third day of Christmas my true love gave to me, three French Hens, " +
-                "two Turtle Doves, and a Partridge in a Pear Tree.\n";
+                "two Turtle Doves, " + "and a Partridge in a Pear Tree.\n";
```

Comment:
I think you missed one of the `+` @FridaTveit mentioned :slightly_smiling_face:

Reply:
Oops. Thanks for catching that!